catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/forms/SearchFormsLoader.java

```diff
@@ -282,6 +282,9 @@ private void createSystemMetacards(List<Metacard> systemTemplates) {
       try {
         catalogFramework.create(new CreateRequestImpl(dedupedTemplateMetacards));
       } catch (SourceUnavailableException | IngestException e) {
+        // The wrapped exception stacktrace isn't shown when the forms are loaded from the
```

Comment:
❓ Why does one message says `load` while the other says `create`? Shouldn't they be consistent?

Reply:
The `RuntimeException` message gets printed when `forms:load` fails (hence the generic load failure message) but this log is localized to the `catalogFramework.create` call so I made it more specific. I can make them the same if you prefer though.